notebooks/124-hugging-face-hub/124-hugging-face-hub.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>probably need to provide info about torchscript=True </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done!</p>